app/services/analytics_events.rb

```diff
@@ -2791,5 +2791,13 @@ def contact_redirect(redirect_url:, step: nil, location: nil, flow: nil, **extra
   def show_password_button_clicked(path:, **extra)
     track_event('Show Password Button Clicked', path: path, **extra)
   end
+
+  # Tracks if a user clicks the 'acknowledge' checkbox during personal
+  # key creation
+  # @param [boolean] checked whether the user checked or un-checked
+  #                  the box with this click
+  def personal_key_acknowledgment_toggled(checked:, **extra)
```

Comment:
Most (all?) other "IdV:"-prefixed events also prefix the method name `idv_`

```suggestion
  def idv_personal_key_acknowledgment_toggled(checked:, **extra)
```